### PR TITLE
Prefix to postfix in cpp

### PR DESCRIPTION
**Pull request by aaadddiii**
Fixes issue #1819

### CODE REVIEW

**Comment by aaadddiii** [comment]
@nakul-19 @raksha009 @plazzy99 @tarun26091999 @rish-singhal please review it

File: Stack/readme.md [hunk 1]
```diff
@@ -29,3 +29,4 @@ element in a full stack is called **overflow**.
 -   Reverse individual words of a string ----> [C++](/Code/C++/reverse_words_of_string.cpp)
 -   Stack Class ----> [C++](/Code/C++/stack_class.cpp)
 -   Stock Span Problem ----> [C++](/Code/C++/Stock_Span_Problem.cpp) | [Java](/Code/Java/Stock_Span_Problem.Java) | [Python](/Code/Python/StockSpan.py)
```
Comment (rish-singhal) [inline Stack/readme.md:31]: Sort this list.

File: Code/C++/Prefix_to_postfix.cpp [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    cin >> exp;
+    prefixtopostfix(exp);
+    return 0;
+}
+
+/*
+Sample I/O:
+
+Enter the prefix expression
+*+abc
```
[the remark this answers is not in the document]
Comment (rish-singhal) [reply Code/C++/Prefix_to_postfix.cpp:55]: Note.

File: Code/C++/Prefix_to_postfix.cpp [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+// if operand function checks if the element is an operand
+int ifoperand(char c)
+{
+    if ((c >= 'a' && c <= 'z') || (c >= 'A' && c <= 'Z'))
+    {
+        return 1;
+    }
+    return 0;
+}
+// prefixtopostfix function converts a prefix expression to postfix
```
Comment (rish-singhal) [inline Code/C++/Prefix_to_postfix.cpp:17]: Add empty line here

File: Code/C++/Prefix_to_postfix.cpp [hunk 4]
```diff
@@ -0,0 +1,60 @@
+/*
+An expression is called prefix , if the operator appears in the expression before the operands. (operator operand operand)
+An expression is called postfix , if the operator appears in the expression after the operands . (operand operand operator)
+The program below accepts an expression in prefix and outputs the corresponding postfix expression .
+*/
+#include <bits/stdc++.h>
+using namespace std;
+// if operand function checks if the element is an operand
```
Comment (rish-singhal) [inline Code/C++/Prefix_to_postfix.cpp:8]: Add empty line here

**Review by rish-singhal — CHANGES_REQUESTED** [review]

**Pull request closed without merging**